Sources/Target/Linux/Process.cpp

```diff
@@ -102,6 +105,48 @@ static pid_t blocking_waitpid(pid_t pid, int *status, int flags) {
   return ret;
 }
 
+ErrorCode Process::readMemory(Address const &address, void *data, size_t length,
+                              size_t *count) {
+#if defined(HAVE_PROCESS_VM_READV)
+  struct iovec local_iov = {data, length};
+  struct iovec remote_iov = {reinterpret_cast<void *>(address.value()), length};
+  auto id = _currentThread == nullptr ? _pid : _currentThread->tid();
+
+  ssize_t ret = process_vm_readv(id, &local_iov, 1, &remote_iov, 1, 0);
+  if (ret >= 0) {
+    *count = ret;
+    return kSuccess;
+  } else {
```

Comment:
nit: you don't need the else (but a religion thing so you can leave it if you want)